podpac/core/compositor.py

```diff
@@ -81,20 +82,51 @@ class Compositor(Node):
     source = tl.Unicode().tag(attr=True)
     sources = ArrayTrait(ndim=1)
     cache_native_coordinates = tl.Bool(True)
-
     interpolation = interpolation_trait(default_value=None)
+    strict_source_outputs = tl.Bool(False)
```

Comment:
@mpu-creare Please let me know if you would like to use a different name for this. I also considered `strict_outputs_checking` and a number of others.

I also considered flipping the boolean and calling it `combine_outputs`, which will coincide with the similar `CombineOutputs` algorithm node.

We can also enable strict checking by default, if you would prefer.